lib/bolt/secret/base.rb

```diff
@@ -36,7 +36,7 @@ def secret_decrypt(opts)
       def validate_resolve_reference(opts)
         # TODO: Remove deprecation warning
         if opts.include?('encrypted-value')
-          raise Bolt::ValidationError, "The 'encrypted-value' key is deprecated migrate to to 'encrypted_value'"
+          raise Bolt::ValidationError, "Inventory file parsing error: The 'encrypted-value' key in the inventory file is deprecated and can no longer be used. Please change the name of the key to 'encrypted_value' instead, and all will be fine."
```

Comment:
It is not really an inventory error. Would `Plugin error` help make this easier to understand?

Reply:
From the user's perspective, it's not clear what a `Plugin error` is....
The problem is that the inventory file contains a key that's deprecated and needs to be changed. The error message should reflect that, if it is to be of help to the user.

Reply:
these values will be used from bolt.yaml too so it's not just inventory.

Reply:
Ah I see, ok that warrants some changes